cosmic-applet-audio/src/pulse.rs

```diff
@@ -219,6 +219,7 @@ impl PulseHandle {
                 loop {
                     // This is where the we match messages from the GUI to pass to the pulse server
                     let mut msgs = Vec::new();
```

Comment:
Perhaps this vec could be moved above the loop and drained when iterated so it doesn't reallocate.

Reply:
Being mindful to truncate of course